todo_app/data/todomapper.py

```diff
@@ -0,0 +1,21 @@
+class ToDoMapper():
```

Comment:
This class seems a little messy. First of all these are all class attributes rather than instance, so there's no need to create instances of the ToDoMapper.

But really it's just a collection of constants. Or you could maybe use an enum.

Also, rather than having all of Trello's field names stored together here, would it be neater to split them up so that `Item`, `BoardStatus` and `Board` know about the field names relevant to themselves?

Reply:
Agreed - removed completed and moved these to each respective class